.github/workflows/create-release.yml

```diff
@@ -0,0 +1,61 @@
+name: "Create release"
+
+on:
+  workflow_dispatch:
+    inputs:
+      milestone_title:
+        description: 'Milestone title'
+        required: true
+
+env:
+  MAIN_BRANCH: "master"
+  RELEASE_BRANCH: "release"
+
+jobs:
+  create-changelog-PR:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Checkout spinalcordtoolbox
+      uses: actions/checkout@v2
+      with:
+        ref: ${{ env.RELEASE_BRANCH }}
+
+    - name: Install spinalcordtoolbox
+      run: |
+        ./install_sct -iy
+
+    - name: Update requirements-freeze.txt
+      run: |
+        source python/etc/profile.d/conda.sh
+        conda activate venv_sct
+        pip freeze | grep -v "-e git+" | grep -v "torch" > requirements-freeze.txt
+        conda deactivate
+        echo "# Platform-specific torch requirements (See SCT Issue #2745)" >> requirements-freeze.txt
+        grep "torch" requirements.txt >> requirements-freeze.txt
+
+    - name: Update version.txt
+      run: |
+        echo "${{ github.event.inputs.milestone_title }}" > spinalcordtoolbox/version.txt
+
+    - name: Commit and push
+      run: |
+        git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
+        git config --global user.name "GitHub Actions Bot"
+        git fetch
+        git checkout ${{ env.MAIN_BRANCH }}
+        git checkout ${{ env.RELEASE_BRANCH }}
+        git merge --no-ff ${{ env.MAIN_BRANCH }}
+        git add requirements-freeze.txt
+        git commit -m "Update requirements-freeze.txt for ${{ github.event.inputs.milestone_title }}"
+        git add spinalcordtoolbox/version.txt
+        git commit -m "Update version.txt for ${{ github.event.inputs.milestone_title }}"
+        git push
```

Comment:
Now that I think about it, `release` is currently a "protected branch", yet it has no protections applied to it:

<details>
![Screenshot from 2021-05-09 10-53-17](https://user-images.githubusercontent.com/16181459/117576689-ce7f8e80-b0b4-11eb-8019-4ca28ab09434.png)
</details>

I'm wondering if I'll still need to provide some sort of authentication for these actions (personal access token, etc.). It worked OK on my personal repo, though. Hrm.